packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -59,6 +57,24 @@ Tests can be executed in any of the following supported form factors:
 | huge     | 1300  | 768    |
 | enormous | 1500  | 768    |
 
+## Environment Variables
+
+The following environment variables are configurable for the test run. Variables marked as reserved will be overwritten by the test runner. To configure reserved variables use the test runner configuration options. If the test runner is not being used the variables are configurable by the user.
+
+| Variable Name                 | Type    | Reserved | Default   | Description                                                                                                                               |
+|-------------------------------|---------|----------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| BROWSERS                      | string  | true     |           | A list of browsers for the test run. Browsers are delimited by a comma. Example: chrome,firefox,ie                                        |
```

Comment:
Technically by default this is chrome & firefox correct?

Reply:
There's two defaults, locally (not on the grid) it's only chrome, if the grid is enabled it's chrome, firefox, and ie

Reply:
I added chrome as the default:

https://github.com/cerner/terra-toolkit/pull/544/commits/545286169a5e7ce61f25b37444a1105be829b750